src/_data/books.yml

```diff
@@ -76,3 +76,10 @@
   cover: programming-flutter.jpg
   link: "https://pragprog.com/book/czflutr"
   desc: "Native, Cross-Platform Apps the Easy Way. Doesn't require previous Dart knowledge."
+- title: 'Flutter Libraries we love'
```

Comment:
These books are organized alphabetically by title. Please move this up in the list.

Reply:
Yeah, true.